internal/integratedservices/services/backup/manager.go

```diff
@@ -0,0 +1,69 @@
+// Copyright © 2021 Banzai Cloud
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package backup
+
+import (
+	"context"
+
+	"github.com/banzaicloud/integrated-service-sdk/api/v1alpha1/backup"
+
+	"github.com/banzaicloud/pipeline/internal/integratedservices"
+)
+
+// Manager component implementing input / output manipulation as required for the backup integrated service
+type Manager struct {
+	version string
+}
+
+func NewManager(version string) Manager {
+	return Manager{
+		version: version,
+	}
+}
+
+func (m Manager) GetOutput(ctx context.Context, clusterID uint, spec integratedservices.IntegratedServiceSpec) (integratedservices.IntegratedServiceOutput, error) {
+	return map[string]interface{}{
+		"externalDns": map[string]interface{}{
+			"version": m.version,
+		},
+	}, nil
+}
```

Comment:
**Question**: is this reference right? 
Shouldn't this be related to the output of the backup/velero service? (similar to [internal/integratedservices/services/backup/outputresolver.go:29](https://github.com/banzaicloud/pipeline/pull/3475/files#diff-13a0f2cfdd754f137df8a27a0bbc344157e8147fcbf8f17191c11a9439022898R29) )

Reply:
fixed